web/src/index.js

```diff
@@ -88,7 +88,7 @@ function isUserSignedIn() {
 async function saveMessage(messageText) {
   // Add a new message entry to the Firebase database.
   try {
-    await addDoc(collection(getFirestore(), "messages"), {
+    await addDoc(collection(getFirestore(), "chatrooms/"+selectedRoom+"/messages"), {
```

Comment:
Would you mind running `prettify` (formatter for JS) on this doc? I think there should be spaces in between the + symbols here and in the other calls to `collection()`, and maybe some spaces after commas. No worries if you don't have that set up either, I can run it after too

Alternatively, if you're doing JS string formatting, you can replace "chatrooms/"+selectedRoom+"/messages" with \`chatrooms/${selectedRoom}/messages\`

Reply:
```suggestion
    await addDoc(collection(getFirestore(), `chatrooms/${selectedRoom}/messages`), {
```